packages/venia-ui/lib/components/Icon/icon.css

```diff
@@ -10,3 +10,10 @@
     fill: var(--fill, none);
     stroke: var(--stroke);
 }
+
+@media (min-width: 641px) {
+    .icon_desktop {
```

Comment:
We want to avoid using classnames for coincidental aspects of presentation, such as this one. Classnames should semantically distinguish one element from another (`root` vs `icon`) or, optionally, one logical state from another (`expanded` vs `collapsed`).

Fortunately, it's irrelevant in this case since icons should be the same size on both mobile and desktop (`24px` here). Let's remove this classname and the places that target it.

Reply:
Removed icon_desktop and its usage